airbyte-integrations/connector-templates/source-generic-jdbc/.gitignore.hbs

```diff
@@ -0,0 +1 @@
+NEW_SOURCE_CHECKLIST.md
```

Comment:
this .gitignore is no longer needed since we don't generate the `NEW_SOURCE_CHECKLIST` file

Reply:
removed, thanks